starwars/src/App.js

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import './App.css';
+import StarWars from './components/StarWars'
```

Comment:
Need to change this to Starwars since your file name has the lowercase 'w'. It only loads after fixing that